### ISSUE TITLE
Mongoose findOneAndModify deprecation warning (low priority)

### ISSUE THREAD

**Issue by joshmorel** [FightPandemics/FightPandemics#842]
We get this warning where we are using `findOneAndUpdate` 
```
(node:29) DeprecationWarning: Mongoose: `findOneAndUpdate()` and `findOneAndDelete()` without the `useFindAndModify` option set to false are deprecated. See: https://mongoosejs.com/docs/deprecations.html#findandmodify
```

We should use `    useFindAndModify: false,` option

**Comment by kaydeejay** [comment]
I'd like to fix this bug

**Comment by kaydeejay** [comment]
I've added the `{ useFindAndModify: false }` option to the query in `backend/lib/endpoints/posts.js`, but now when I try to 'like' a post on the UI, the request seems to go through successfully but the like icon does not change. Oddly enough, if I load the page and the post is already liked, I can 'unlike' it, but not 'like' it again. [Demonstration](https://imgur.com/PskFmAM)

**Comment by joshmorel** [comment]
Hey Kevin - this issue actually seems to be related to
________________________________

**Comment by joshmorel** [comment]
Resolved by #1095
